### PR TITLE
CX-2696 Fix/clean build script

### PR DESCRIPTION
**Pull request by rfreitas**
# Problem

Some build configurations were redundant and some unnecessary confusing.

# Solution

Removed unneeded indirection between the actual NODE_ENV and the readable NODE_ENV.
Moved woopra domain to CONFIG value.
Made config values automatically part of the readable ENV variables.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [n/a] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a ] Have new automated tests been implemented?
- [n/a ] Have new manual tests been written down?
- [n/a ] Have tests passed locally?
- [n/a ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-492-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-492-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://492-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-492-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-492-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://492-pr-onfido-sdk-ui-onfido.surge.sh

File: webpack.config.babel.js [hunk 1]
```diff
@@ -107,9 +105,13 @@ const PROD_CONFIG = {
   'MOBILE_URL' : 'https://id.onfido.com',
   'SMS_DELIVERY_URL': 'https://telephony.onfido.com',
   'PUBLIC_PATH' : `https://assets.onfido.com/web-sdk-releases/${packageJson.version}/`,
+  WOOPRA_DOMAIN
 }
 
-const TEST_CONFIG = { ...PROD_CONFIG, PUBLIC_PATH: '/', 'MOBILE_URL' : '/' }
+const TEST_CONFIG = { ...PROD_CONFIG,
```
Comment (hjheath) [inline webpack.config.babel.js]: Not introduced in this PR, but It worries me that we are using the same config for TEST and PROD environments.

Comment (rfreitas) [reply webpack.config.babel.js]: the same endpoints, yes, but not the same config per se.
I agree in principle, but practically staging causes too much flakiness

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-492-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-492-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://492-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-492-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://492-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-492-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**